Core/Security/VOMS.py

```diff
@@ -8,8 +8,7 @@
 from DIRAC.Core.Security.BaseSecurity import BaseSecurity
 from DIRAC.Core.Security.X509Chain import X509Chain
 from DIRAC.Core.Utilities.Subprocess import shellCall
-from DIRAC.Core.Utilities import List
-from DIRAC.Core.Utilities import Os
+from DIRAC.Core.Utilities import List, Time, Os
```

Comment:
Although we don't follow strictly pep8, check out https://www.python.org/dev/peps/pep-0008/#id17 :beers: